### PR TITLE
[Backport] Backend: Fixed checkbox alignment

### PR DESCRIPTION
**Pull request by SyntaxKay**
### Original Pull Request 
 https://github.com/magento/magento2/pull/19631

### Description (*)

Checkbox alignment issue backend. 
### Fixed Issues (if relevant)

1. magento/magento2#19630: Checkbox alignment issue backend. 

### Manual testing scenarios (*)

1. Login into admin panel > System > Export > checkbox alignment issue "Fields Enclosure".

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @speedy008. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.2-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Review by rodrigowebjump — COMMENTED** [review]
Hi @speedy008 

Why did you not use the same variable as in the original PR?
What is the reason you modify the test?

**Comment by SyntaxKay** [comment]
@rodrigowebjump I have created Back-port PR using porting tool. I will check with Original PR nd update it soon.

File: dev/tests/functional/tests/app/Magento/Catalog/Test/Block/Adminhtml/Product/Edit/Section/Options.php [hunk 1]
```diff
@@ -402,4 +402,21 @@ public function getFileOptionElements()
     {
         return $this->_rootElement->getElements($this->hintMessage);
     }
+
+    /**
+     * @inheritdoc
+     */
+    protected function _fill(array $fields, SimpleElement $element = null)
```
Comment (rogyar) [inline dev/tests/functional/tests/app/Magento/Catalog/Test/Block/Adminhtml/Product/Edit/Section/Options.php:409]: Could you provide more information about this change, please? I'm not sure it should be included as a part of the backport. Correct me if I'm wrong. Thank you

**Review by rogyar — CHANGES_REQUESTED** [review]

**Comment by rogyar** [comment]
Thank you for your collaboration. Starting from Feb 19 minor style fixes are considered as "Cleanup" and no longer accepted for 2.2-develop branch.

**Comment by Ctucker9233** [comment]
@rogyar Please explain this logic. Is 2.2 at EOL?

**Pull request closed without merging**